packages/wallets/src/features/cfd/constants.tsx

```diff
@@ -128,7 +128,7 @@ export const CFD_PLATFORMS = {
 export const MARKET_TYPE = {
     ALL: 'all',
     FINANCIAL: 'financial',
-    SYNTHETIC: 'synthetic',
+    STANDARD: 'standard',
```

Comment:
We do not have such a thing as standard market type. Kindly change it back to synthetic